### PR TITLE
docs: Fix typos, remove unnecessary semicolons and blank lines

### PR DESCRIPTION
**Pull request by kangetsu121**
- Fix typos in MDX and some codes.
- Remove unnecessary semicolons and blank lines. This PR removes extra semicolons appearing in docs.

### CODE REVIEW

File: www/src/pages/components/placeholder.mdx [hunk 1]
```diff
@@ -46,7 +46,7 @@ within the element to reflect the height when actual text is rendered in its pla
   The use of <code>aria-hidden="true"</code> only indicates that the element
   should be hidden to screen readers. The <i>loading</i> behaviour of the
   placeholder depends on how authors will actually use the placeholder styles,
-  how they plan to update things, etc. Some JavasSript code may be needed to{' '}
+  how they plan to update things, etc. Some JavasScript code may be needed to{' '}
```
Comment (golota60) [inline www/src/pages/components/placeholder.mdx:49]:
```suggestion
  how they plan to update things, etc. Some JavaScript code may be needed to{' '}
```

**Review by golota60 — APPROVED** [review]
lgtm, one small nit

**Comment by kangetsu121** [comment]
@golota60 
Thank you for your suggestion, I committed you change as 8af16a9.

**Comment by golota60** [comment]
Thanks!

**Pull request merged**